addons/mail/static/src/js/basic_view.js

```diff
@@ -17,7 +17,8 @@ BasicView.include({
             hasRecordReloadOnAttachmentsChanged: post_refresh === 'always',
             hasRecordReloadOnMessagePosted: !!post_refresh,
             hasRecordReloadOnFollowersUpdate: !!followers_post_refresh,
-            isActivityBoxVisible: this._getFieldOption('message_ids', 'open_attachments', false),
+            isActivityBoxVisible: this._getFieldOption('message_ids', 'open_attachments', false)
```

Comment:
1/ Why ? 
2/ Should be done in a separate commit, in which we explain why, so that I don't have to ask it :D